Code/PgSQL/rdkit/mol_op.c

```diff
@@ -182,9 +182,9 @@ mol_substruct_count(PG_FUNCTION_ARGS) {
   PG_RETURN_##ret( func(i) );                                           \
 }
   
-MOLDESCR(amw,MolAMW,FLOAT4)
-MOLDESCR(logp,MolLogP,FLOAT4)
-MOLDESCR(tpsa,MolTPSA,FLOAT4)
+MOLDESCR(amw,(float)MolAMW,FLOAT4)
```

Comment:
Just to silence a bunch of warnings with MSVC.

On Thu, Feb 4, 2016 at 9:06 AM, Greg Landrum <notifications@github.com>
wrote:

> In Code/PgSQL/rdkit/mol_op.c
> <https://github.com/rdkit/rdkit/pull/752#discussion_r51845104>:
>
> > @@ -182,9 +182,9 @@ mol_substruct_count(PG_FUNCTION_ARGS) {
> >    PG_RETURN_##ret( func(i) );                                           \
> >  }
> >
> > -MOLDESCR(amw,MolAMW,FLOAT4)
> > -MOLDESCR(logp,MolLogP,FLOAT4)
> > -MOLDESCR(tpsa,MolTPSA,FLOAT4)
> > +MOLDESCR(amw,(float)MolAMW,FLOAT4)
>
> I'm sure I'm missing something obvious, but it's not clear to me why all
> of these casts to float are required.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rdkit/rdkit/pull/752/files#r51845104>.
>